### ISSUE TITLE
Lorawan yaml

### ISSUE THREAD

**Issue by rallep71** [PaulSchulz/esphome-lora-sx126x#4]
Hello, I want to test lorawan now, but I can't get any further because the yaml can't be compiled in esphome.
here is my yaml, and the error message.

`Updating /config/esphome/lorawan.yaml
------------------------------------------------------------

INFO ESPHome 2024.10.3
INFO Reading configuration /config/esphome/lorawan.yaml...
ERROR Error while reading config: Invalid YAML syntax:

while parsing a block collection
  in "/config/esphome/lorawan.yaml", line 15, column 3
expected <block end>, but found '?'
  in "/config/esphome/lorawan.yaml", line 19, column 3
=========== [ERROR] /config/esphome/lorawan.yaml ===========

======================== [SUMMARY] ========================
  - /config/esphome/lorawan.yaml: FAILED`
  
yaml:

`esphome:
  name: lorawan-sx126x
  friendly_name: lorawan
  libraries:
     - "SPI"
     - "Ticker"
     - "SX126x-Arduino"

esp32:
  board: esp32-s3-devkitc-1
  framework:
    type: arduino

external_components:
  - source:
    type: git
    url: https://github.com/PaulSchulz/esphome-lora-sx126x
    ref: main
  components: ["lorawan_sx126x"]

# Enable logging
logger:

wifi:
  networks:
  - ssid: !secret wifi_ssid
    password: !secret wifi_password
  
api:
  encryption:
    key: "2JYEzqRQzDRVPxFmSMFpy99eejls6IeXmH+GOEyo6Tk="

ota:
  - platform: esphome
    password: "a1038de807cc488795133cf7fcd4de32"

#
web_server:
  port: 80

#spi:
#  clk_pin: GPIO9
#  mosi_pin: GPIO10
#  miso_pin: GPIO11

#i2c:
#  - id: bus_a
#    sda: 17
#    scl: 18

# LoRaWAN Specific Oprions
# Setting these will enable the LoRaWAN radio mode
lorawan_sx126x:`

**Comment by PaulSchulz** [comment]
My apologies. LoRaWAN is not yet supported.

**Comment by Jaishankar872** [comment]
is P2P communication possible? (Reference [link](https://github.com/beegee-tokyo/SX126x-Arduino/issues/126))

**Comment by PaulSchulz** [comment]
Yes, via LoRa. Let me check and I will get back to you. Do you need
bidirectional? Another option might be Meshtastic.. which uses RadioLib and
supports a lot more boards than I do.

**Comment by Jaishankar872** [comment]
Yes, bidirectional for home automation. Like sending a comment control device and then the device will send an acknowledgement reply message.
